### ISSUE TITLE
Unholy DK APL does not Outbreak at the onset of an add wave, resulting in subpar Epidemics.

### ISSUE THREAD

**Issue by Hekili** [simulationcraft/simc#5859]
### Describe the bug

Unholy DK APL does not Outbreak at the onset of an add wave, resulting in subpar Epidemics.

### Expected behavior

Ideally, if targets are going to live long enough to pop Epidemic a few times, you would want to get Virulent Plague on many targets so that ~~Fire Nova~~ Epidemic damage is maximized.

### To Reproduce
Please include any steps necessary to recreate the problem, or include logs/data to show what is occurring in game if there is an issue with how something is modeled.

Sim in Hectic Add Cleave.  Outbreak is not used when new adds arrive.

I added a pretty basic line to the APL to trigger that Outbreak to see the results.  I haven't attempted to further optimize, but there's probably more that could be done (i.e., identifying the breakpoint for how many new VPs are needed to be worth the GCD).  

The result, as-is, is a 13.6% DPS gain.

![image](https://user-images.githubusercontent.com/9438134/111926529-857f6680-8a83-11eb-8391-2cb58446f964.png)

https://www.raidbots.com/simbot/report/iamwEAPAfWGjSxSwVPkdSK

```
actions+=/outbreak,if=dot.virulent_plague.refreshable&!talent.unholy_blight&!raid_event.adds.exists
actions+=/outbreak,if=dot.virulent_plague.refreshable&active_enemies>=2&(!talent.unholy_blight|talent.unholy_blight&cooldown.unholy_blight.remains)
actions+=/outbreak,if=runeforge.superstrain&(dot.frost_fever.refreshable|dot.blood_plague.refreshable)
+ actions+=/outbreak,line_cd=5,if=active_enemies-active_dot.virulent_plague>2
```

### Additional information
* SimulationCraft Version (required)

321cf96cb6

* If SimulationCraft is crashing, please include your **dxdiag** file - Instructions here - 
  https://us.battle.net/support/en/article/Obtaining-System-Files

N/A

* **simc_gui.simc** file or contents
  This file is saved in the same folder as SimulationCraft after each simulation. Make sure to run the 
  simulation that is giving you issues, then include the file here by either uploading to github, 
  copy/pasting the contents, or using http://pastebin.com/

N/A

**Comment by nyterage** [comment]
PR open with a fix here. 
https://github.com/simulationcraft/simc/pull/5861

Issue looks like it was an issue with parsing `dot.virulent_plague.refreshable` on an if condition. Only checked the main target for the DoT being refreshable, swapping it to a `target_if` condition fixes the line that was intended to do that already.

**Comment by Armadk** [comment]
Fixed in #5861 thanks!
